modules/auxiliary/scanner/http/caidao_bruteforce_login.rb

```diff
@@ -0,0 +1,82 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'metasploit/framework/credential_collection'
+require 'metasploit/framework/login_scanner/caidao'
+
+class Metasploit4 < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::AuthBrute
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Chinese Caidao Backdoor Bruteorce',
+      'Description'    => 'This module attempts to brute chinese caidao asp/php/aspx backdoor.',
+      'Author'         => [ 'Nixawk' ],
+      'References'     => [
+        ['URL', 'https://www.fireeye.com/blog/threat-research/2013/08/breaking-down-the-china-chopper-web-shell-part-i.html'],
+        ['URL', 'https://www.fireeye.com/blog/threat-research/2013/08/breaking-down-the-china-chopper-web-shell-part-ii.html'],
+        ['URL', 'https://www.exploit-db.com/docs/27654.pdf'],
+        ['URL', 'https://www.us-cert.gov/ncas/alerts/TA15-313A'],
+        ['URL', 'http://blog.csdn.net/nixawk/article/details/40430329']
+      ],
+      'License'        => MSF_LICENSE
+    ))
+
+    register_options(
+      [
+        OptString.new('LOGIN_URL', [true, 'The URL that handles the login process', '/caidao.php']),
```

Comment:
Instead of using LOGIN_URL, wouldn't it be a good idea to use TARGETURI for the sake of consistency?